.travis.yml

```diff
@@ -1,7 +1,21 @@
 language: go
 
-script: go test -v ./...
-
 go:
-  - "1.9.x"
   - "1.10.x"
+
+env:
+  - PROTOBUF_VERSION=3.5.1
+
+before_install:
+  - basename=protoc-${PROTOBUF_VERSION}-linux-x86_64
+  - wget https://github.com/google/protobuf/releases/download/v${PROTOBUF_VERSION}/$basename.zip
+  - unzip -d /home/travis $basename.zip
+  - rm $basename.zip
+  - PATH=/home/travis/bin:$PATH protoc --version
+  - make install
```

Comment:
Do you need make install.
The idea of downloading the release is that you can skip make install?

Reply:
Make install installs the generate dependencies in the vendor folder.

Reply:
Ah my bad